setup.cfg

```diff
@@ -24,3 +24,4 @@ console_scripts =
     copy-resources-s3 = mbtools.copy_resources_s3:main
     fetch-os-pages = mbtools.fetch_openstax_pages:main
     validate-mbz-html = mbtools.validate_mbz_html:main
+    replace-content-tags = mbtools.html_tag_insertion:main
```

Comment:
This looks like it may be incorrect. It could be `replace-content-tags = mbtools.html_content_extractor:main`, but perhaps we should rename `replace-content-tags`. The other commands seem to take `{verb}-{something}`, so maybe this should be `extract-html-contents` (and perhaps a similar name change to `html_contents_extractor.py` --> `extract_html_contents.py`)